### ISSUE TITLE
64 Snapshots Oversize Plugin Question

### ISSUE THREAD

**Issue by MasterChiefJon** [alanrenouf/vCheck-vSphere#640]
I have v1.4 of this plugin.  I see at the beginning of the script that you initialize the variable 
$snapp = @()
but I don't see where you add any data to the variable anywhere in the script.  Then at the end, it contains data (and I do get output showing my VMs with snapshots.  
How does the $snapp variable get populated with data?

**Comment by MasterChiefJon** [comment]
On line 22, the plugin creates a "New-Object".    This object contains the information about all of the snapshots.  I modified this line to this:
`$snapp += New-Object -TypeName PSObject -Property ([ordered]@{
And now the new-object information is saved into the $snapp variable.

**Comment by Sneddo** [comment]
Yeah, the way it currently works is by passing the data back down the pipeline (and thus not using that variable). However, that will mean we don't do any ordering...
